webpack.config.js

```diff
@@ -57,6 +57,10 @@ module.exports = {
           compilation.errors.push(new Error(
             `Detected ${numCyclesDetected} cycles which exceeds configured limit of ${MAX_CYCLES}`
           ));
+        } else if (numCyclesDetected > 0) {
```

Comment:
Depending on the time frame for this work, this could be noisy. I'll take this to indicate that you expect to be finished soon :)

Reply:
Yep. I want it to be done and dusted before I disappear. Besides, this is one line. It's already followed by a line for each cycle it finds, so it's only incrementally a little noisy. When the count goes to zero, I'll remove it.